recommender/engines/rl/training/data_extraction_step/data_extraction_step.py

```diff
@@ -44,26 +44,26 @@ def __init__(self, pipeline_config):
 
     def __call__(self, data=None) -> Tuple[List[Sars], dict]:
         start = time()
-        sarses = self._generate_proper_sarses()
+        sarses = self._generate_sarses()
         end = time()
-        return sarses, {"sarses_generation_duration": end - start}
+        return sarses, {"sarses_generation_duration": f"{end - start}s"}
```

Comment:
I would format this to only log 2 decimal places

Reply:
It's funny that I did exactly that in #322 PR. It will be merged. https://github.com/cyfronet-fid/recommender-system/blob/e6e62a01a137117876591f043e17f23441cafcb5/recommender/engines/rl/training/data_extraction_step/data_extraction_step.py#L49